src/closestPair.js

```diff
@@ -0,0 +1,34 @@
+export default function setUnion( setOfPoints = [] ){
+  if( setOfPoints == null ){
+    return 'Cannot find closest pair of null values'
+  }
+  if( setOfPoints.constructor !== Array ){
+    return 'Please use Arrays'
+  }
+
+  let point1, point2
+
+  const bruteForce = _ => {
+    let min = Infinity
```

Comment:
Whoops forgot to change name of that function setUnion to closestPair, seems test still passes though good catch. Also wasn't sure if I want to keep the namespace of bruteForce limited by keeping it in closestPair? Or does it not matter since we're only exporting closestPair anyway? What's the convention when it comes to minimizing global namespace usage?